QUICKSTART.md

```diff
@@ -21,35 +21,45 @@ Table of contents:
 - [3. Teardown the environment](#3-teardown-the-environment)
 
 
-This guide is intended to show how to quickly bring up a **KubeStellar** environment with its dependencies from a binary release.
+This guide is intended to show how to (1) quickly bring up a **KubeStellar** environment with its dependencies from a binary release and then (2) run through a simple example usage.
 
 ## 1. Install and run **KubeStellar**
 
-KubeStellar works in the context of kcp, so to use KubeStellar you also need kcp. Download the kcp and **KubeStellar** binaries and scripts into a `kubestellar` subfolder in your current working directory using the following command:
+KubeStellar works in the context of kcp, so to use KubeStellar you
+also need kcp. The first command below will (a) download unpack kcp
+and KubeStellar if they are not already on your `$PATH` and (b) launch
+them if they are not already running.  This command is more fully
+described in [its
+documentation](https://docs.kubestellar.io/docs/coding-milestones/poc2023q1/commands/#quick-setup).
```

Comment:
this link appears to be dead already? :( It would be nice if this mentioned that the overview architecture diagram could be found there too. That was what I was really missing when I first read through the quickstart. Just 'what' am I deploying and where/how. I can very much appreciate keeping the quickstart short for those that don't want to know those details, but it would have helped me a bunch more to understand how the components i'm about to deploy  interact. including what's in docker, what's not in docker, what's in the k8s cluster etc. That's certainly too detailed to put in a quickstart, but just pointing me at it here would be great